### DISCUSSION TITLE
API: top_score always null

### DISCUSSION THREAD

**Discussion by SHU-red** [blakeblackshear/frigate#14696] [Ask A Question] [answered]
Am i the only one seeing at `GET/api/events` always a `top_score` `null`?

Dont know much about it and maybe misconfigured my frigate but i recognized not getting any telegram notifications by using `telegram-notify` with `min-score` setting and not getting any notifications since setting it to e.g. `75`

And i also saw that API returns `top_score` but API doku shows min/max score

Does all of this come down to a big api change maybe

1. Not updated in api documentation
2. Not updated in tools using the api?

Below a /api/events example from my current frigate 0.14.1-f4f3cfa
```yaml
[{"box":null,"camera":"garage","data":{"attributes":[],"box":[0.428125,0.1425925925925926,0.37864583333333335,0.8527777777777777],"region":[0.3104166666666667,0.0,0.6145833333333334,1.0925925925925926],"score":0.9731574058532715,"top_score":0.9655845165252686,"type":"object"},"end_time":1730320228.789495,"false_positive":null,"has_clip":true,"has_snapshot":true,"id":"1730320212.849335-3ck4p9","label":"person","plus_id":null,"retain_indefinitely":false,"start_time":1730320207.849335,"sub_label":null,"thumbnail":"nonono","top_score":null,"zones":[]}]
```

**Comment by hawkeye217** [comment, marked as answer]
`top_score` at the root level of the response is deprecated and will return a value for older events only. You should use the `top_score` inside the `data` object.

**Reply by SHU-red** [reply]
Thank you for your quick reply
I will open up a issue in the repo of telegram-notify
